_sass/minima/_layout.scss

```diff
@@ -5,6 +5,9 @@
   font-size: $small-font-size;
   text-align: center;
   padding-top: 3 * $spacing-unit;
+  padding-bottom: $spacing-unit;
+  z-index: 1;
+  background-color: white;
```

Comment:
`background-color: $white;` 🥇

Reply:
I am assuming you're saying I should change `white` to `$white`. will do :+1:

Reply:
Done. Amended the existing commit to fix this.